controller/apps.go

```diff
@@ -351,7 +350,7 @@ func (c *AppsController) ShowSpace(ctx *app.ShowSpaceAppsContext) error {
 	// get OpenShift space
 	space, err := kc.GetSpace(*kubeSpaceName)
 	if err != nil {
-		return witerrors.NewInternalError(ctx, err)
+		return witerrors.NewInternalError(ctx, errs.Wrap(err, "could not retrieve space "+*kubeSpaceName))
```

Comment:
Please use `errs.Wrapf(err, "could not retrieve space: %s", *kubeSpaceName)` instead.

Reply:
Sure, but why is the more inefficient form is preferred?  Readability?

Reply:
I didn't know it was more inefficient.

Reply:
Everything I've read on the web says so.  String concatenation gives the compiler a chance to optimize; apparently the Go compiler will now combine concatenations to make fewer object allocations, for instance.  Since this is all error handling it really makes no difference to performance, but in the past I've used concatenation where I can (Also, c++ iostreams makes it sort of natural). 
I'm happy to use Errorf() for readability and/or consistency.

Reply:
@stooke thanks for the explanation. C++ ISO streams are a bit controversially discussed in the community, but okay... 
I did use `Errorf` because of readability but feel free to do it as you think.